src/typeahead/typeahead.ts

```diff
@@ -108,6 +108,11 @@ export class NgbTypeahead implements OnInit,
   @Input() showHint: boolean;
 
   /**
+  * Auto select the first item in the result list
+  */
+  @Input() autoSelectItem: boolean = null;
```

Comment:
I don't think you need initialization here, especially to `null`. I think it defaults to `true` in the config